packages/ui/src/hooks/use-theme-mode.ts

```diff
@@ -89,9 +91,17 @@ const useSyncMode = (onChange: (mode: ThemeMode) => void) => {
 };
 
 /**
- * Sets the give value in local storage
+ * Sets the give value in local storage.
+ * "auto" will remove the local storage value if present.
  */
-const setModeInLS = (mode: ThemeMode) => localStorage.setItem(LS_THEME_MODE, mode);
+const setModeInLS = (mode: ThemeMode) => {
+  if (mode === "auto") {
```

Comment:
this will break the current logic, `auto` value is meant to be saved in LS and from that computed to either `light` or `dark`